lib/main.coffee

```diff
@@ -11,16 +11,8 @@ module.exports =
       type: 'string'
       title: 'PHPMD Rulesets'
       default: 'cleancode,codesize,controversial,design,naming,unusedcode'
-      description: 'Comma separated list of rulesets to use in phpmd.'
+      description: 'Comma separated list of rulesets to use in phpmd. You can also enter the name of your ruleset file (example: `ruleset.xml`) to load that from the current file\'s directory (or any of the parent directories) '
       order: 2
-    projectRules:
```

Comment:
Like I said in the previous comment I'm not removing the feature itself, but suggesting it's merged into the rulesets setting.